### PR TITLE
Add documentation for V3

### PR DESCRIPTION
**Pull request by tir38**
This is documentation for V3 and V3 for Cash App Pay.

### CODE REVIEW

**Review by BrianGardnerAtl — APPROVED** [review]
Looks good to me. The setup instructions were clear. Not sure if we'd want to more strongly recommend folks go with a V3 implementation so there's no server communication required

**Comment by tir38** [comment]
> Looks good to me. The setup instructions were clear. Not sure if we'd want to more strongly recommend folks go with a V3 implementation so there's no server communication required

Confusingly the `V` in `V3` is not "version".  `V3` is not an upgrade over the other "variations". All three are different ways to use Afterpay SDK.

File: docs/getting-started/checkout-v3.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Checkout V3 is currently available in the following region(s): US, UK, Australia and Canada
+> 
+Checkout V3 generates a one-time payment card for every Afterpay order and provides the card number to insert into your credit card checkout. This allows for a front-end-only integration. Unlike V1 and V2, with V3 your server does not directly interact with Afterpay APIs. The one-time payment card is Visa for the US, UK, and Canada. In Australia the one-time payment card is Mastercard.
```
Comment (mmroz) [inline docs/getting-started/checkout-v3.md:24]: This is added to the alert section, I think we can leave it as plain text

File: docs/getting-started/v2-with-cash-app-pay.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-title: Cash App Pay
+title: Checkout V2 with Cash App Pay
```
Comment (mmroz) [inline docs/getting-started/v2-with-cash-app-pay.md:3]: On iOS this is just Cash App Pay

File: docs/getting-started/v3-with-cash-app-pay.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
+---
+layout: default
+title: Checkout V3 with Cash App Pay
+parent: Getting Started
+nav_order: 7
```
Comment (mmroz) [inline docs/getting-started/v3-with-cash-app-pay.md:5]: On iOS this is before Cash App Pay

File: docs/getting-started/checkout-v3.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                        exception: Exception?,
+                    ),
+                ->
+                Log.e(tag, cancellationStatusV3.toString(), exception)
+            }
+    }
+}
+```
+
+## Step 4: Receive one-time use card details and pass back to your server for processing
```
Comment (mmroz) [inline docs/getting-started/checkout-v3.md:91]: I don't think you need to pass these back to your server, you can drop these into any checkout form

File: docs/getting-started/checkout-v3.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+The success result will contain card details, tokens, and a valid-until time. Pass these back to your own server and process them through your normal card processing infrastructure.
+
+```kotlin
+Afterpay.parseCheckoutSuccessResponseV3(intent)?.let {
+  // pass card details and token to your server for processing with
+  // your existing card processing network
+  it.cardDetails
+  it.tokens
+  it.cardValidUntil
+```
```
Comment (mmroz) [inline docs/getting-started/checkout-v3.md:102]: If the SDK supports `updateMerchantReferenceV3` we should add an entry

File: docs/getting-started/checkout-v3.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Checkout V3
+parent: Getting Started
+nav_order: 5
+---
+
+# Checkout V3
```
Comment (mmroz) [inline docs/getting-started/checkout-v3.md:8]: iOS has a Sequence Diagram, you can probably use the same one

File: docs/getting-started/v3-with-cash-app-pay.md [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+
+            result.onFailure {
+             // TODO
+            }
+        }
+    }
+}
+```
+
+## Step 11: Receive one-time use card details and pass back to your server for processing
```
Comment (mmroz) [inline docs/getting-started/v3-with-cash-app-pay.md:304]: We don't need to send details back to server

File: docs/getting-started/v3-with-cash-app-pay.md [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+```
+
+## Step 12: Unregister
+
+You should also use the **Unregister** function when you're done with the SDK:
+
+``` kotlin
+cashAppPay.unregisterFromStateUpdates()
+```
+
```
Comment (mmroz) [inline docs/getting-started/v3-with-cash-app-pay.md:323]: You can include the sequence diagram here

**Review by mmroz — APPROVED** [review]

File: docs/getting-started/v2-with-cash-app-pay.md (hunk 2 above)
Comment (tir38) [reply docs/getting-started/v2-with-cash-app-pay.md:3]: I want to distinguish v2 and v3 each with cash app pay

File: docs/getting-started/checkout-v3.md (hunk 5 above)
Comment (tir38) [reply docs/getting-started/checkout-v3.md:102]: I'll add this to my TODO

File: docs/getting-started/checkout-v3.md (hunk 6 above)
Comment (tir38) [reply docs/getting-started/checkout-v3.md:8]: I removed it on purpose. I think is not accurate and a bit confusing.

File: docs/getting-started/v3-with-cash-app-pay.md (hunk 7 above)
Comment (tir38) [reply docs/getting-started/v3-with-cash-app-pay.md:304]: they need to do *something* with those card details right?

**Pull request merged**